manifest.webapp

```diff
@@ -63,6 +63,11 @@
       "description": "Required by the cozy-bar to display Claudy and know which applications are coming soon",
       "type": "io.cozy.settings",
       "verbs": ["GET", "PUT"]
+    },
+    "organizations": {
+      "description": "Required to share passwords with other persons",
+      "type": "com.bitwarden.organizations",
+      "verbs": ["GET", "POST"]
```

Comment:
Did we add these permissions to the store? https://docs.cozy.io/en/cozy-stack/doctype/#add-your-doctype-to-the-store-and-stack

Reply:
Not yet, I started to edit cozy-store a few days ago, but since we merged organizations and collections doctypes, so I still have to reflect those changes before pushing to github

Reply:
Here it is: https://github.com/cozy/cozy-store/pull/741